assets/model_monitoring/components/src/model_data_collector_preprocessor/mdc_preprocessor_helper.py

```diff
@@ -67,65 +67,56 @@ def same_year(start, end) -> List[str]:
         if is_same_month(start, end):
             return same_month(start, end)
         if is_start_of_year(start) and is_end_of_year(end):
-            return [get_url(f"{start.strftime('%Y')}/*/*/*/*.jsonl")] \
-                if store_url.is_folder_exists(f"{start.strftime('%Y')}") else []
+            return _get_files(f"{start.strftime('%Y')}/*/*/*/*.jsonl")
         return cross_month(start, end)
 
     def same_month(start, end) -> List[str]:
         if is_same_day(start, end):
             return same_day(start, end)
         if is_start_of_month(start) and is_end_of_month(end):
-            return [get_url(f"{start.strftime('%Y/%m')}/*/*/*.jsonl")] \
-                if store_url.is_folder_exists(f"{start.strftime('%Y/%m')}") else []
```

Comment:
is_folder_exists still be used? If not, should we remove it?

Reply:
not used by this scenario, but it's a useful StoreUrl utils for future use, so I choose to keep it there.